lib/elixir/lib/system.ex

```diff
@@ -205,7 +205,13 @@ defmodule System do
       }
 
   """
-  @spec build_info() :: %{required(atom) => String.t()}
+  @spec build_info() :: %{
+          build: String.t(),
+          date: String.t(),
+          revision: String.t(),
+          version: String.t(),
+          otp_release: String.t()
+        }
```

Comment:
Actually, maybe we should define this as a type that we refer to from this spec?

Reply:
sorry, I don't get it

Reply:
Like this. Not sure what's preferable, let's wait for other comments.

```elixir
@type build_info() :: %{...}
@spec build_info() :: build_info()
```

Reply:
I see. Well, if we are not calling it from any other function, I think we shouldn't yet